src/tools/neorados.cc

```diff
@@ -400,3 +406,5 @@ int main(int argc, char* argv[])
 
   return 0;
 }
+#pragma GCC diagnostic pop
```

Comment:
Nit: I'd expect a blank line here since it's covering the entire file:
```suggestion

#pragma GCC diagnostic pop
```